src/test/test_controller.c

```diff
@@ -1689,6 +1691,187 @@ test_download_status_bridge(void *arg)
   return;
 }
 
+/** Mock cached consensus */
+static cached_dir_t *mock_ns_consensus_cache;
+static cached_dir_t *mock_microdesc_consensus_cache;
+
+int
+mock_we_want_to_fetch_flavor(const or_options_t *options,
+                             int flavor);
+
+int
+mock_we_want_to_fetch_flavor(const or_options_t *options,
+                             int flavor) {
+  if (options->FetchUselessDescriptors ||
+      flavor == FLAV_NS || flavor == FLAV_MICRODESC) {
+    return 1;
+  }
+  return 0;
+}
+
+/**  Mock the function that retrieves consensus from
```

Comment:
Seems like you are wrapping text at like 60 columns, whereas we usually wrap at 80. It seems even more weird than our already weird wrapping logic, so please adjust all your code to fit the norm.